packages/apps/composer-app/src/containers/ComposerDocument.tsx

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import React from 'react';
+
+import type { Item } from '@dxos/client';
+import { Composer, DOCUMENT_TYPE } from '@dxos/react-composer';
+import { TextModel } from '@dxos/text-model';
+
+export const COMPOSER_DOCUMENT = DOCUMENT_TYPE;
+
+export const ComposerDocument = ({ item }: { item: Item<TextModel> }) => {
```

Comment:
since this component does nothing but apply specific styles (and doesn't marry with any app specific schema) it is therefore "pure" and is not a container (imo) :)

Reply:
Yeah, I was wondering about this… `react-composer` _is_ the container, and it doesn’t export any pure components.

It looks like `tasks-app` no longer uses `react-list`; should we also refactor `react-composer` into `composer-app`’s containers?

Reply:
I went ahead and deleted `containers`, including `ComposerDocument`.